staffSchedules/models.py

```diff
@@ -1,14 +0,0 @@
-from django.db import models
-
-from employees.models import Employee
-
-
-# Create your models here.
-class staffSchedule(models.Model):
```

Comment:
need to add class Meta
add relevant helping + error messages
first character of the class name should be capital